custom_components/google_home/translations/da.json

```
@@ -23,7 +23,7 @@
     "step": {
       "init": {
         "data": {
-          "update_interval": "Ændre opdaterings interval. Hæv dette vis du lider af timeout fejl. Standard: 180 (Sekunder)"
+          "update_interval": "Ændre opdaterings interval. Hæv dette Hvis du lider af timeout-fejl. Standard: 180 (Sekunder)"
```

i guess this does not have to be capitilised?
```suggestion
          "update_interval": "Ændre opdaterings interval. Hæv dette hvis du lider af timeout-fejl. Standard: 180 (Sekunder)"
```

---

Correct. I was a bit too fast fixing the spelling mistakes. Oops. It's not supposed to be capital H in that spot :)